web/src/components/njwds-extended/UnStyledButton.tsx

```diff
@@ -41,6 +41,9 @@ export const UnStyledButton = forwardRef(
         style =
           "usa-button usa-tag bg-transparent text-normal text-base border-1px border-base-light hide-unhide-button";
         break;
+      case "basic":
+        style = "usa-button bg-transparent";
+        break;
```

Comment:
So I tried using all 3 of the other styles outlined in this component first and ended up overriding multiple other things like the border or the hide-unhide-button or the usa-button--unstyled and some of those end up being hard to override because they're !important. So in the end this ended up being the most intuitive to understand

Although I'm open to name changes besides "basic"

Reply:
The goal of this case is to make an unstyled variant, correct? Should we call it that instead of `basic`?

I see what you're saying about `.usa-button--unstyled`. Not something you introduced, but I do find it funny that the unstyled class has an !important color.

Reply:
Lol, like I said I'm down for a different name and that seems good. 

Yeah kinda absurd that unstyled has a !important color. Some BS right there